### PR TITLE
added Linkedin Link

### LINKED ISSUE #320: [FEATURE] UPDATE FOOTER FOR GOOD-FIRST-ISSUE-FINDER
## Description

It would be better to add Twitter and Linked in links also 

### Screenshots

Screenshot (33)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by nithin999989**
closes #320 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by nithin999989** [comment]
@Cahllagerfeld

**Comment by nithin999989** [comment]
@eddiejaoude 
@Cahllagerfeld i updated linked in link please merge my pull request

**Comment by Cahllagerfeld** [comment]
So you want to work on this PR, and close #326 in favor of this one correct?

File: src/lib/components/footer.svelte [hunk 1]
```diff
@@ -29,6 +29,17 @@
           aria-label="Github"><i class="fa-brands fa-github fa-xl" title="Github" /></a
         >
       </li>
+      <li>
+  <a
+    data-test-id="linkedin-btn"
+    target="_blank"
+    rel="noreferrer"
+    class="transition duration-200 hover:text-[#333333] active:text-primary-100 dark:hover:text-[#fafafa]"
```
[the remark this answers is not in the document]
Comment (nithin999989) [reply src/lib/components/footer.svelte]: yes i made a commit and updated that

**Comment by nithin999989** [comment]
@eddiejaoude please review my code

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -29,6 +29,17 @@
           aria-label="Github"><i class="fa-brands fa-github fa-xl" title="Github" /></a
         >
       </li>
+      <li>
+  <a
+    data-test-id="linkedin-btn"
+    target="_blank"
+    rel="noreferrer"
+    class="transition duration-200 hover:text-[#0e76a8] active:text-primary-100 dark:hover:text-[#fafafa]"
```
Comment (Cahllagerfeld) [inline src/lib/components/footer.svelte:37]: We can also remove `dark:hover:text....` as we don't need a dedicated style for dark theme (unlike the github one with low contrast).

**Review by Cahllagerfeld — COMMENTED** [review]
Besides that change it looks good to me 👍

**Review by Cahllagerfeld — APPROVED** [review]
Nice work!

**Pull request merged**